Bull's Eyes Game/Bull's Eyes Game/ViewController.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  ViewController.swift
+//  Bull's Eyes Game
+//
+//  Created by User-P2 on 4/1/23.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
```

Comment:
You need put access modified like final to this class:
```suggestion
final class ViewController: UIViewController {
```